src/Linkification/Embedding.coffee

```diff
@@ -119,6 +120,22 @@ Embedding =
         $.extend data.link, <%= html('[${data.key}] <span class="warning">Title Link Blocked</span> (are you using NoScript?)</a>') %>
     return
 
+  preview: (data) ->
+    {key, uid, link} = data
+    service = Embedding.types[key].preview
+    $.on link, 'mouseover', (e) ->
+      src = service uid
+      el = $.el 'img',
+        src: src
+        id: 'ihover'
+      $.add d.body, el
+      UI.hover
+        root: link
+        el: el
+        latestEvent: e
+        endEvents: 'mouseout click'
+        asapTest: -> el.height
```

Comment:
I would assume they are, considering youtube is adding black borders to the images.